cel/syntax.pb.go

```diff
@@ -514,7 +514,7 @@ type SourceInfo struct {
 	//
 	// The line number of a given position is the index `i` where for a given
 	// `id` the `line_offsets[i] < id_positions[id] < line_offsets[i+1]`. The
-	// column may be derived from `id_positions[id] - line_offsets[i]`.
+	// column may be derivd from `id_positions[id] - line_offsets[i]`.
```

Comment:
Unfortunate typos, but we should probably just let this through since anyone who tries to generate in the future will keep regenerating this change.